pkg/controller/clusterdeprovision/clusterdeprovision_controller_test.go

```diff
@@ -196,8 +207,9 @@ func TestClusterDeprovisionReconcile(t *testing.T) {
 func testClusterDeprovision() *hivev1.ClusterDeprovision {
 	return &hivev1.ClusterDeprovision{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      testName,
-			Namespace: testNamespace,
+			Name:        testName,
+			Namespace:   testNamespace,
+			Annotations: map[string]string{},
```

Comment:
Just curious, why was this change added?  Were we missing Annotations before?

I don't see you using them anywhere else in this PR.

Reply:
Ah I had a test for the approval annotation prior which was removed, this is still in. I'll kill it.